### ISSUE TITLE
Timeout errors relating to openpolicyagent

### ISSUE THREAD

**Issue by marcus-bcl** [ministryofjustice/cloud-platform#4379]
## Service name

HMPPS Probation Integration Services

## Service environment

- [x] Dev / Development
- [x] Staging
- [x] Prod / Production
- [ ] Other

## Impact on the service

Intermittently impacts CI only, deployments fail with timeout errors but usually succeed on retry.

## Problem description

We're intermittently getting timeout errors relating to `openpolicyagent` on Ingress, when performing a few deployments in parallel.  This happens a few times per week.

Here's an example: https://github.com/ministryofjustice/hmpps-probation-integration-services/actions/runs/4132642698/jobs/7141865543
```
Error: UPGRADE FAILED: release manage-pom-cases-and-delius failed, and has been rolled back due to atomic being set: 
cannot patch "manage-pom-cases-and-delius-v1-2" with kind Ingress: 
Internal error occurred: failed calling webhook "webhook.openpolicyagent.org": 
Post "[https://opa.opa.svc:443/?timeout=10s](https://opa.opa.svc/?timeout=10s)": context deadline exceeded
```

## Contact person

Marcus Aspin - [Slack](https://mojdt.slack.com/archives/D75HJEU3B)
Probation Integration Team - [#probation-integration-tech](https://mojdt.slack.com/archives/C02HQ4M2YQN)

**Comment by vijay-veeranki** [comment]
Placed this ticket on hold, and requested Marcus to upgrade the helm version they are using and try it again.

**Comment by vijay-veeranki** [comment]
Closing this one, will wait for the user to come back if there are any more errors.
